infrastructure/running-a-validator/operational-tasks/upgrade-your-node.md

```diff
@@ -1,85 +1,68 @@
-
-This guide will provide an option for validators to seamlessly substitute an active validator server to allow for maintenance operations. The process can take several hours, so make sure you understand the instructions first and plan accordingly.
-
-!!!tip 
-    Keep an eye out for new releases from the community and upgrade or downgrade accordingly.
+This guide will allow validators to seamlessly substitute an active validator server to allow for maintenance operations. The process can take several hours, so ensure you understand the instructions first and plan accordingly.
 
 ## Prerequisites
 
-Before continuing with this guide, ensure you have:
-
-<!--TODO: what should I have already read/done/installed before I go on? -->
-
-- A fully working validator setup, including all necessary binaries in accordance to [How to Validate](todo:add-link)
-- The capacity on your VPS to start a secondary validator for the upgrade process
+Before beginning the upgrade process for your validator node, ensure the following:
 
-## Key Components
+- You have a fully functional validator setup with all required binaries installed. See [Set Up a Validator](infrastructure/running-a-validator/onboarding-and-offboarding/set-up-validator.md) and [Validator Requirements](polkadot-docs/infrastructure/running-a-validator/requirements.md) for additional guidance
+- Your VPS infrastructure has enough capacity to run a secondary validator instance temporarily for the upgrade process
 
-### Session Keys
+## Session Keys
 
-Session keys are stored in the client and used to sign validator operations. They are what link your
-validator node to your staking proxy. If you change them within a session you will have to wait for
-the current session to finish and a further two sessions to elapse before they are applied.
+Session keys are used to sign validator operations and establish a connection between your validator node and your staking proxy account. These keys are stored in the client, and any change to them requires a waiting period. Specifically, if you modify your session keys, the change will take effect only after the current session is completed and two additional sessions have passed.
 
-Visit the [Keys section](https://wiki.polkadot.network/docs/learn-cryptography#keys){target=\_blank} of the Polkadot Wiki to learn more about the types of keys and their usage in the Polkadot network.
+Remembering this delayed effect when planning upgrades is crucial to ensure that your validator continues to function correctly and avoids interruptions. To learn more about session keys and their importance, check out the [Keys section](https://wiki.polkadot.network/docs/learn-cryptography#keys){target=\_blank} in the Polkadot Wiki.
```

Comment:
I'm not sure if this page will still exist on the Wiki and we don't have session keys documented on our cryptography page. Just flagging this now 🤷‍♀️

Reply:
I opened a ticket to follow up on this. Changed it to a generic "To learn more about session keys and their importance, visit the [Keys section](TODO: update-path){target=\_blank}." until we figure it out.